src/macad_gym/viz/render.py

```diff
@@ -25,8 +27,9 @@ def multi_view_render(images, unit_dimension, actor_configs):
     for actor_id, im in images.items():
         if not actor_configs[actor_id]["render"]:
             continue
-        surface = pygame.surfarray.make_surface(im.swapaxes(0, 1) * 128 + 128)
-        surface_seq += ((surface, (poses[actor_id][1], poses[actor_id][0])), )
+        im = cv2.resize(im, (unit_dimension[1], unit_dimension[0]))
+        surface = pygame.surfarray.make_surface(im)
+        surface_seq += ((surface, (poses[actor_id][0], poses[actor_id][1])), )
```

Comment:
What's the motivation behind changing the index order from: `(poses[actor_id][1], poses[actor_id][0])` to `(poses[actor_id][0], poses[actor_id][1])`?

Reply:
Okay. The order is stay consistent with PyGame's (x, y) ordering and the config `x_res` & `y_res` convention.
For square resolutions (x == y) that we use, the (x, y) and (y, x) doesn't effect a change. I'll mark this as resolved.